otter/worker/launch_server_v1.py

```diff
@@ -566,17 +568,16 @@ def prepare_launch_config(scaling_group_uuid, launch_config):
     server_config['metadata'].update(generate_server_metadata(
         scaling_group_uuid, launch_config))
 
-    if server_config.get('name'):
-        server_name = server_config.get('name')
-        server_config['name'] = '{0}-{1}'.format(server_name, generate_server_name())
-    else:
-        server_config['name'] = generate_server_name()
+    suffix = generate_server_name()
```

Comment:
Non-blocking nitpick: `generate_server_name` name confuses a little with `prepare_server_name`. Would you be fine with renaming `generate_server_name` to `generate_random_server_name`?